impl.go

```diff
@@ -83,18 +83,31 @@ func findInterface(iface string, srcDir string) (path string, id string, err err
 	if err != nil {
 		panic(err)
 	}
-	if len(f.Imports) == 0 {
+
+	qualified := strings.ContainsRune(iface, '.')
```

Comment:
ContainsByte?

Reply:
There is no `ContainsByte` that I know of. Do you prefer `ContainsAny`, or `IndexByte`, or something else over `ContainsRune`?

Reply:
Huh. Let’s use Contains(iface, “.”).